ament_clang_format/package.xml

```diff
@@ -9,6 +9,11 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>python3-setuptools</build_depend>
+
+  <buildtool_depend>python3</buildtool_depend>
```

Comment:
> Imo this should be handled somewhere else in the release pipeline.

Do you have a suggestion as to where? Should bloom magically inject python build deps when ament_package is the build type.
Should I just patch package.xmls at release time.